judging-algorithm/formJsonOutput.ts

```diff
@@ -0,0 +1,35 @@
+import { FinalOutputTables, JudgeOutput, JudgeOutputLiveSite, JudgeOutputProjectsLiveSite } from "./types";
+import fs from "fs";
+
+/* Creates JSON file for the front-end hackers page to consume for judging assignments */
+export function convertHackersTablesToJson(
+    finalOutputTables: FinalOutputTables,
+  ): void {
+    const hackerTables = JSON.stringify(finalOutputTables.hackerOutput);
+    fs.writeFileSync('hackerResults.json', hackerTables);
+}
+
+/* Creates JSON files for the front-end judges page to consume for judging assignments */
+export async function convertJudgesTablesToJson(
+    judgeStrings: string[],
+    finalOutputTables: FinalOutputTables,
+  ): Promise<void> {
+    const judgeTables = JSON.stringify(judgeDataLiveSite(judgeStrings, finalOutputTables.judgeOutput));
+    fs.writeFileSync('judgeResults.json', judgeTables);
+}
+  
+/* Massage JudgeOutput structure data in to the live site structure */
+function judgeDataLiveSite(judgeStrings: string[], judgeOutputs: JudgeOutput[]): JudgeOutputLiveSite[] {
+    let judges: JudgeOutputLiveSite[] = [];
+    for (var judge of judgeStrings) {
```

Comment:
nit: forEach would be more idiomatic here, but in reality looks like you're doing a map right